Guided Investigation - Incident Triage.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.                Assingned to: {incident['properties.owner.userPrincipalName']} - 
</pre><p>typo - Assigned to</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/120/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>